src/main/java/com/alinab/taskOneGOFpatterns/creational/factoryMethod/factory/DesignerFactory.java

```diff
@@ -0,0 +1,24 @@
+package com.alinab.taskOneGOFpatterns.creational.factoryMethod.factory;
+
+import com.alinab.taskOneGOFpatterns.creational.factoryMethod.designers.Designer;
+import com.alinab.taskOneGOFpatterns.creational.factoryMethod.designers.DesignerType;
+import com.alinab.taskOneGOFpatterns.creational.factoryMethod.designers.GraphicalDesigner;
+import com.alinab.taskOneGOFpatterns.creational.factoryMethod.designers.InteriorDesigner;
+
+public class DesignerFactory {
+
+    public Designer createDesigner(DesignerType designerType) {
+        Designer designer = null;
+
+        switch (designerType) {
+            case GRAPHICAL_DESIGNER:
+                designer = new GraphicalDesigner();
+                break;
+            case INTERIOR_DESIGNER:
+                designer = new InteriorDesigner();
+                break;
+        }
```

Comment:
What if we will have some other type? Add default with exception

Reply:
Done